### DISCUSSION TITLE
数字类型显示错误

### DISCUSSION THREAD

**Discussion by 1415749952** [dbeaver/dbeaver#11815] [DBeaver Development]
The field number (6, 2) is defined in Oracle database. The value of this field is 60.00. In dbeaver, the query is displayed as 60. Do not show 00 after decimal point

**Comment by LonwoLonwo** [comment]
Hello @1415749952 
This is default behavior.
You can change it in Preferences.
Preferences -> Editors -> Data Editor -> Data Formats
Then choose type "Numbers" and "Minimum fraction digits"

![2021-03-23 12_41_03-DBeaver Enterprise 21 0 1 - 10274](https://user-images.githubusercontent.com/45152336/112126250-51b45600-8bd5-11eb-97d9-a7b682ada878.png)

![2021-03-23 12_41_31-DBeaver Enterprise 21 0 1 - 10274](https://user-images.githubusercontent.com/45152336/112126257-537e1980-8bd5-11eb-92b6-7ba6e7d92ed7.png)
